src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/index.jsx

```diff
@@ -60,18 +69,42 @@ export default function ConfirmedAppointmentDetailsPage() {
 
   useEffect(
     () => {
-      const pageTitle = isCommunityCare ? 'Community care' : 'VA';
+      let pageTitle;
+      let prefix = null;
+
+      if (isPast) prefix = 'Past';
+      else if (isCanceled) prefix = 'Canceled';
+
+      if (isCommunityCare)
+        pageTitle = prefix ? `${prefix} community care` : 'Community care';
+      else if (isInPerson) {
+        if (appointment?.vaos?.isCompAndPenAppointment)
+          pageTitle = prefix ? `${prefix} claim exam` : 'Claim exam';
+        else pageTitle = prefix ? `${prefix} in-person` : 'In-person';
+      } else if (isVideo) pageTitle = prefix ? `${prefix} video` : 'Video';
+      else if (isVAPhoneAppointment) {
+        pageTitle = prefix ? `${prefix} phone` : 'Phone';
+      }
+
       const pageTitleSuffix = featureBreadcrumbUrlUpdate
         ? ' | Veterans Affairs'
         : '';
       if (appointment && appointmentDate) {
         document.title = `${pageTitle} appointment on ${appointmentDate.format(
-          'dddd, MMMM D, YYYY',
+          'dddd, MMMM DD, YYYY',
         )}${pageTitleSuffix}`;
         scrollAndFocus();
       }
     },
-    // [appointment, appointmentDate, isCommunityCare, featureBreadcrumbUrlUpdateæ],
+    [
+      appointment,
+      appointmentDate,
+      isCommunityCare,
+      isCanceled,
+      isInPerson,
+      isPast,
+      featureBreadcrumbUrlUpdate,
```

Comment:
Hmm linter is complaining that `isVideo` is missing from this list.

Reply:
Done.